python/BUILDING.md

```diff
@@ -16,13 +16,50 @@ important differences:
 
     `$ make python_wheel`
 
-After this procedure completes, the `ngraph/build/python/dist` directory should 
-contain the Python packages of the version you cloned. For example, if you 
-checked out and built `0.21` for Python 3.7, you might see something like: 
+After this procedure completes, the `ngraph/build/python/dist` directory should
+contain the Python packages of the version you cloned. For example, if you
+checked out and built `0.21` for Python 3.7, you might see something like:
 
     $ ls python/dist/
-    ngraph-core-0.21.0rc0.tar.gz  
-    ngraph_core-0.21.0rc0-cp37-cp37m-linux_x86_64.whl  
+    ngraph-core-0.21.0rc0.tar.gz
+    ngraph_core-0.21.0rc0-cp37-cp37m-linux_x86_64.whl
+
+## Building nGraph Python Wheels on Windows
+
+It's now possible to build the python wheel on Windows too. The build process consists of 3 steps:
+
+1. Configure the build with the following `cmake` invocation:
+~~~~
+cmake ..
+      -G"Visual Studio 15 2017 Win64"
+      -DCMAKE_BUILD_TYPE=Release
+      -DCMAKE_INSTALL_PREFIX="C:\temporary_install_dir"
+      -DNGRAPH_DEX_ONLY=TRUE
+      -DNGRAPH_PYTHON_BUILD_ENABLE=TRUE
+      -DNGRAPH_ONNX_IMPORT_ENABLE=TRUE
+      -DCMAKE_CXX_COMPILER=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\VC\Tools\MSVC\14.16.27023\bin\Hostx64\x64
+~~~~
+There are a couple of things to notice here. One is that the full path to the x64 version of
+MSVC compiler has to be specified. This is because DNNL requires a 64-bit version and cmake might
+not detect it correctly by itself.
+On Windows only the DEX mode is supported - hence the `NGRAPH_DEX_ONLY` flag.
```

Comment:
Only the DEX mode is supported on Windows systems, hence the `NGRAPH_DEX_ONLY` flag.